### ISSUE TITLE
Block tab cycling through forms from breaking stepper

### ISSUE THREAD

**Issue by michael-small** [Megabittron/DeploymentExperiment-URS#215]
This is an evident problem when trying to rapid fire move through the fields on the submission stepper, particularly on the second step. I typically hit tab and press down a letter to fill in the title/abstract, but this can introduce a bug if this is done while also tabbing into a dropdown or radio button. 

Some letters when spammed while tabbed over a dropdown picks an option. When everything is supposedly filled, you cannot advance to the next tab using the 'Next' button. Most of the time this can be fixed by manually clicking an option in the dropdown, even if it's the same option.

However, there is another layer to the bug beyond this. Sometimes the dropdown "fix" isn't even sufficient: this may also apply to the radio buttons, but unlike the dropdowns, I have yet to find a way to refresh the state of the radio buttons to fix this. 

Bottom line: tabbing through options should be viable. Whatever causes this bug must be addressed.

**Comment by michael-small** [comment]
ADDITION: Also applies to the Presenter/Discipline/Featured stepper.
